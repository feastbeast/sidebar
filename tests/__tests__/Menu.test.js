import React from 'react';
import Enzyme from 'enzyme';
import {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Menu from '../../client/src/components/Menu';

Enzyme.configure({ adapter: new Adapter() });

describe('Menu', () => {
  it('should be defined', () => {
    let menu_url = 'google.com';

    let wrapper = shallow(<Menu
      menu_url={menu_url} 
    />) 
    expect(wrapper).toBeDefined();
  });
  it('should render correctly', () => {
    let menu_url = 'google.com';

    let tree = shallow(<Menu
      menu_url={menu_url} 
    />) 
    expect(tree).toMatchSnapshot();
  });

});