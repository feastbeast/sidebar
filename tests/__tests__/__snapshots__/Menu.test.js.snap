// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render correctly 1`] = `
<a
  target="_blank"
>
  <div
    className="btns"
  >
    <span
      className="buttonText"
    >
        VIEW MENU  
    </span>
    <Icon
      className="icons menuIcon"
      right={true}
    >
      restaurant
    </Icon>
  </div>
</a>
`;
